README.md

```diff
@@ -1,35 +1,16 @@
-# Minimal Starter with Angular on both Server and Browser Platforms
-
 <p align="center">
   <img src="https://cloud.githubusercontent.com/assets/1016365/10639063/138338bc-7806-11e5-8057-d34c75f3cafc.png" alt="Universal Angular" height="320"/>
 </p>
 
 # Angular Universal Starter [![Universal Angular](https://img.shields.io/badge/universal-angular2-brightgreen.svg?style=flat)](https://github.com/angular/universal)
 > Server-Side Rendering for Angular
 
-A minimal Angular starter for Universal JavaScript using TypeScript and Webpack
+A minimal Angular starter for Universal JavaScript using TypeScript using `@angular/cli` or `webpack`
 
 > If you're looking for the Angular Universal repo go to [**angular/universal**](https://github.com/angular/universal)  
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
 
-
-## Installation
```

Comment:
can you mention why there are two folders here and link to each readme

Reply:
Done
Let me know if there's any other issues